shared/src/crypto_jwt.rs

```diff
@@ -204,14 +220,23 @@ async fn verify_with_extended_header<M: Msg + DeserializeOwned>(req: &Parts, tok
         msg: custom_without,
         sig: sig.to_string()
     };
+    req.extensions
+        .remove::<ProxyLogger>()
+        .expect("Should be set by middleware")
+        .send(msg_signed.get_from().clone())
+        .expect("Reciever still lives in middleware");
```

Comment:
I don't fully understand Axum middlewares, are those errors handleable or is the `expect` justified?